https-everywhere-checker

```diff
@@ -1 +1 @@
-Subproject commit 2ec94f1ce11a2046408a8df228447e2f0dd85cda
```

Comment:
FWIW, I'm guessing that using an incorrect version of https-everywhere-checker is why you are getting Travis test failures.